### PR TITLE
Adding initial doc for IntegrationSink

### PR DESCRIPTION
**Pull request by matzew**
## Proposed Changes 

- first attempt for docs on integration sink CRD

### CODE REVIEW

**Comment by matzew** [comment]
thats the arn 😅

File: docs/eventing/sinks/integration-sink/aws_sqs.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  ```yaml
+  apiVersion: Sinks.knative.dev/v1alpha1
+  kind: IntegrationSink
+  metadata:
+    name: integration-sink-aws-sqs
+    namespace: knative-samples
+  spec:
+    aws:
+      sqs:
+        arn: "arn:aws:s3:::my-queue"
```
[the remark this answers is not in the document]
Comment (matzew) [reply docs/eventing/sinks/integration-sink/aws_sqs.md:26]: thats the arn 😅, @dsimansk 
so, yes intentional

File: docs/eventing/sinks/integration-sink/logger.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# Log Sink
+
+The `IntegrationSink` supports the _Log Sink Kamelet_ that logs all data that it receives, through its `log` property. This sink useful for debugging purposes.
+
+## Log Sink Example
+
+Below is an `IntegrationSink` that logs all data that it receives:
+
+  ```yaml
+  apiVersion: Sinks.knative.dev/v1alpha1
```
Comment (Cali0707) [inline docs/eventing/sinks/integration-sink/logger.md:10]: nit:
```suggestion
  apiVersion: sinks.knative.dev/v1alpha1
```

File: docs/eventing/sinks/integration-sink/aws_sqs.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    ```bash
+    kubectl -n <namespace> create secret generic my-secret --from-literal=aws.accessKey=<accessKey> --from-literal=aws.secretKey=<secretKey>
+    ```
+
+## AWS SQS Sink Example
+
+Below is an `IntegrationSink` to send data to AWS SQS:
+
+  ```yaml
+  apiVersion: Sinks.knative.dev/v1alpha1
```
Comment (Cali0707) [inline docs/eventing/sinks/integration-sink/aws_sqs.md:18]: nit:
```suggestion
  apiVersion: sinks.knative.dev/v1alpha1
```

File: docs/eventing/sinks/integration-sink/aws_s3.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+    ```bash
+    kubectl -n <namespace> create secret generic my-secret --from-literal=aws.accessKey=<accessKey> --from-literal=aws.secretKey=<secretKey>
+    ```
+
+## AWS S3 Sink Example
+
+Below is an `IntegrationSink` to send data to an Amazon S3 Bucket:
+
+  ```yaml
+  apiVersion: Sinks.knative.dev/v1alpha1
```
Comment (Cali0707) [inline docs/eventing/sinks/integration-sink/aws_s3.md:18]: nit:
```suggestion
  apiVersion: sinks.knative.dev/v1alpha1
```

File: docs/eventing/sinks/integration-sink/README.md [hunk 5]
```diff
@@ -0,0 +1,12 @@
+# Knative Sink for Apache Camel Kamelet integrations
+
+![stage](https://img.shields.io/badge/Stage-alpah-red?style=flat-square)
+![version](https://img.shields.io/badge/API_Version-v1alpha1-red?style=flat-square)
+
+The `IntegrationSink` is a Knative Eventing custom resource supporting selected [_Kamelets_](https://camel.apache.org/camel-k/latest/kamelets/kamelets.html) from the [Apache Camel](https://camel.apache.org/) project. Kamelets allow users to connect to 3rd party system for improved connectivity, they can act as "sources" or as "sinks". Therefore the `IntegrationSink` allows to send data to external systems out of Knative Eventing in the format of CloudEvents. The integration sink is part of the Knative Eventing core installation.
```
Comment (Cali0707) [inline docs/eventing/sinks/integration-sink/README.md:6]:
```suggestion
The `IntegrationSink` is a Knative Eventing custom resource supporting selected [_Kamelets_](https://camel.apache.org/camel-k/latest/kamelets/kamelets.html) from the [Apache Camel](https://camel.apache.org/) project. Kamelets allow users to connect to 3rd party system for improved connectivity, they can act as "sources" or as "sinks". Therefore the `IntegrationSink` allows sending data to external systems out of Knative Eventing in the format of CloudEvents. The integration sink is part of the Knative Eventing core installation.
```

File: docs/eventing/sinks/README.md [hunk 6]
```diff
@@ -132,6 +132,7 @@ The `svc` in `http://event-display.svc.cluster.local` determines that the sink i
 
 | Name                                                                             | Maintainer | Description                          |
 |----------------------------------------------------------------------------------| -- |--------------------------------------|
+| [IntegrationSink](./integration-sink/README.md)                                  | Knative  | Send events to generic event sink    |
```
Comment (Cali0707) [inline docs/eventing/sinks/README.md:135]:
```suggestion
| [IntegrationSink](./integration-sink/README.md)                                  | Knative  | Send events to a generic event sink    |
```

Comment (Cali0707) [reply docs/eventing/sinks/README.md:135]: Also, do we want to link to the specific IntegrationSinks here as well?

Comment (matzew) [reply docs/eventing/sinks/README.md:135]: not sure. yet

**Review by Cali0707 — APPROVED** [review]
/lgtm
/hold in case you want any other reviews, feel free to unhold!

**Pull request merged**